src/bin/ion/commands/generate/model.rs

```diff
@@ -21,6 +20,18 @@ use serde_json::Value;
 pub struct DataModelNode {
     // Represents the name of this data model
     // Note: It doesn't point to the fully qualified name. To get fully qualified name use `fully_qualified_name()` from `AbstractDataType`.
+    // e.g. For a given schema as below:
+    // ```
+    //  type::{
+    //    name: foo,
+    //    type: struct,
+    //    fields: {
+    //      a: int,
+    //      b: string
+    //    }
+    //  }
+    // ```
+    // The name of the data model would be `Foo` where MyType will represent a Java class or Rust struct.
```

Comment:
```suggestion
    // The name of the abstract data type would be `Foo` where MyType will represent a Java class or Rust struct.
```
?

What is `MyType` in this example?

Reply:
My bad this is supposed to be `Foo`, previously I had used the type name as `MyType`. will change that.